### PR TITLE
fix: add MEDDRA id mapping for Disease semantic type

### PR DESCRIPTION
**Pull request by colleenXu**
Addresses https://github.com/biothings/BioThings_Explorer_TRAPI/issues/217

For BTE to use these operations and handle ID resolution correctly: https://github.com/NCATS-Tangerine/translator-api-registry/pull/72

### CODE REVIEW

File: src/config.ts [hunk 1]
```diff
@@ -207,7 +207,8 @@ export const APIMETA: MetaDataItemsObject = {
       GARD: ['mondo.xrefs.gard', 'disease_ontology.xrefs.gard'],
       HP: ['mondo.xrefs.hp'],
       SNOMEDCT: ['mondo.xrefs.sctid', 'umls.snomed.preferred', 'umls.snomed.non-preferred'],
-      NCIT: ['mondo.xrefs.ncit', 'disease_ontology.xrefs.ncit']
+      NCIT: ['mondo.xrefs.ncit', 'disease_ontology.xrefs.ncit'],
+      MEDDRA: ['mondo.xrefs.meddra', 'disease_ontology.xrefs.meddra'],
```
Comment (andrewsu) [inline src/config.ts:211]: will the trailing comma here cause any problems?

Comment (colleenXu) [reply src/config.ts:211]: I don't think so? There's a trailing comma [here](https://github.com/biothings/biomedical_id_resolver.js/blob/e9149760a1361389b27d5b674545f897068a16a6/src/config.ts#L28) and [here](https://github.com/biothings/biomedical_id_resolver.js/blob/e9149760a1361389b27d5b674545f897068a16a6/src/config.ts#L184)....throughout the file really.

I don't know the syntax though...

**Comment by newgene** [comment]
add a test for MEDDRA id @colleenXu

**Comment by colleenXu** [comment]
I've put it in my to-do list as a non-urgent task. I'll contact later for guidance on how to write the test / where to put it

**Comment by colleenXu** [comment]
Note that we have changed to using refactored ID resolver with SRI-based service (dev)...

It's not clear if code like this is used anymore, or what kind of tests need to be done (discussions in https://github.com/biothings/BioThings_Explorer_TRAPI/issues/283 and issues that I will create later)

**Pull request merged**